cfgov/jinja2/v1/_includes/molecules/info-unit-2.html

```diff
@@ -64,11 +64,19 @@
     {%- if has_image %}
         {%- set img = image( value.image.upload, 'original' ) %}
 
-        {# Don't include image alt text if the image is being linked and
-           there's also heading text in the link. #}
-        {%- set img_alt_text = ''
-            if ( link_image_and_heading and has_heading )
-            else image_alt_value( value.image ) %}
+        {%- if not link_image_and_heading %}
+            {# If the image isn't being linked, include its alt text. #}
+            {% set img_alt_text = image_alt_value( value.image ) %}
+        {% elif has_heading %}
+            {# If the image is being linked, and there's a heading, set
+               empty alt text to avoid having duplicate text in the link. #}
+            {% set img_alt_text = '' %}
+        {% else %}
+            {# If the image is being linked, but there's no heading, use
+               the text from the link. #}
+            {% set img_alt_text = value.links[0].text %}
```

Comment:
Shouldn't we check here and see if the image has alt text specified, and then if not, fall back on the link text?

Reply:
No, per team discussion we prefer to keep the two links the same. As I wrote above:

> If we used image-specific alt text here, then we'd have two links going to the same place, but with
different text. Using the link text means that both links are the same. This is consistent with how the carousel links work on the homepage mega menu.

Reply:
Hrm. Yeah. But repeated links are also bad. Damned if you do, damned if you don't, I guess.

Reply:
Plus if the user provided link text it's likely the provided text is more appropriate for hyperlinking than whatever the alt text is.